app/controllers/monthly_bills_controller.rb

```diff
@@ -2,9 +2,10 @@ class MonthlyBillsController < ApplicationController
   before_action :authenticate_user!
 
   def index
-    @authorized_user = MonthlyBill.joins(:user).where(["monthly_bills.user_id = ? OR monthly_bills.user_id = ? OR users.invited_by_id = ?",
-                                                       current_user.id, current_user.invited_by_id, current_user.id])
-    @search = @authorized_user.order(due_date: 'ASC').ransack(params[:q])
+    authorized_user = MonthlyBill.joins(:user).where(
+          ["monthly_bills.user_id = ? OR monthly_bills.user_id = ? OR users.invited_by_id = ?",
```

Comment:
Line is too long. [95/80]<br>Indent the first parameter one step more than the start of the previous line.